R/clear.cache.R

```diff
@@ -65,3 +65,8 @@ clear.cache <- function (...){
         }
 }
 
+# Determine if a variable is cached or not
+.is.cached <- function (variable) {
+        file.exists(file.path(.cache.dir, paste0(variable, ".RData")))
```

Comment:
If you just want to return the `in.cache` status you could replace the body of the function with `.read.cache.info(var)$in.cache` without any if statements. But I think it would make more sense to do it the other way around and make `.read.cache.info` call `.is.cached`. That way `.is.cached` contains exactly the code to determine whether a variable is cached, and `.read.cache.info` can return the additional info necessary info for cache invalidation. There is no need for a new function, is it?